coding-concepts.js

```diff
@@ -7,48 +7,48 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+//console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will log 5
+// b) Verify and explain: when you push and element to an array it will added to the end of the array. and console log the amount elements in the array.
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+//console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: the .lenght method rerurns the amount of element in the string counting the space
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the first "o"
+// b) Verify and explain: the console log is looking for the fourth index in the string which it will be the first "o". Arrays and strings start from index count 0.
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+//console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: error
+// b) Verify and explain: it console logs "Ruby". but i dont understads why, i thought index was a seprate variable apart from the array of languages.
```

Comment:
Bracket notation is used to access the value at the index. We are using bracket notation on the `languages` array, with the `index` variable (which is 1) inside the brackets. This means we are accessing the value at index 1 in the array, which is "Ruby".

Reply:
Thank you i will make sure to review that.